src/inmanta/data/model.py

```diff
@@ -564,3 +564,12 @@ class NoPushTriggerMethod(str, Enum):
 PromoteTriggerMethod = StrEnum(
     "PromoteTriggerMethod", [(i.name, i.value) for i in chain(const.AgentTriggerMethod, NoPushTriggerMethod)]
 )
+
+
+class DryRun(BaseModel):
+    id: uuid.UUID
+    environment: uuid.UUID
+    model: int
+    date: Optional[datetime.datetime]
```

Comment:
@wouterdb Do you know why `date` can be None. In the database schema allows `NULL`, but in practice I think the date is always set. What would it mean for a DryRun not to have a `date`?